testcafeRunner.js

```diff
@@ -1,5 +1,5 @@
 import createTestcafe from 'testcafe';
-import { App } from './app';
+import { App } from './app/app';
```

Comment:
This is bit unfortunate. The app.js is at the same level as testcafeRunner. So the path was right. Why has this had to change? I think this is going to cause confusion when it comes to development. Is there anyway to sort this out so the paths can stay the same for local

Reply:
The change to move app.js and server.js to ./app came when we were working on the admin FE with @kat1906 . It can be done without touching the location of app.js & server.js. If you have a look at the [first commit](https://github.com/nhs-digital-gp-it-futures/buying-catalogue-admin/pull/5/commits/31eb8f6a3dea66ce43d80496d252a548b82628a2) in [this PR](https://github.com/nhs-digital-gp-it-futures/buying-catalogue-admin/pull/5) - that's how it would look like. I'm not opposed to reverting it to the way it was as it makes the commit smaller and cleaner, however I'm sure there's reason why we moved it. We'd need to speak to @kat1906 to tell us what the reason was.